addons/pharma/functions/fnc_inspectBreath.sqf

```diff
@@ -29,4 +30,8 @@ if (_ph > 750) then {
     _output = LLSTRING(breath_good);
 };
 
+if (_hr == 0 || !(alive _patient)) then {
```

Comment:
maybe add obstruction and occlusion too but i think it would require some discussion with other devs